scripts/istat/geos/import_config.txt

```diff
@@ -0,0 +1,41 @@
+import: {
+  import_name: "istat-region"
+  provenance_url: "https://www.istat.it/it/archivio/6789"
+  provenance_description: "ISTAT code for territorial units in Italy"
+  mcf_url: "https://storage.cloud.google.com/resolved_mcf/external_tables/istat-region/nodes.mcf-00000-of-00001"
+  # First date of imported data.
+  start_date: "2020-06-23"
```

Comment:
We can leave the start and end date out.

Reply:
done